src/core/constants/loading-instructions.js

```diff
@@ -57,13 +57,13 @@ const MAP = {
  * Returns the loading instruction with a higher priority. The priority
  * order is auto -> lazy -> eager -> unload.
  *
- * @param {Loading_Enum|string} v1
- * @param {Loading_Enum|string} v2
+ * @param {Loading_Enum|string|undefined} v1
+ * @param {Loading_Enum|string|undefined} v2
```

Comment:
nit: would this (or `=` notation) be more expressive here?
```suggestion
 * @param {Loading_Enum|string} [v1]
 * @param {Loading_Enum|string} [v2]
```